### DISCUSSION TITLE
How get selected style attribute?

### DISCUSSION THREAD

**Discussion by yulafezmesi** [ueberdosis/tiptap#3261] [General Questions]
Hey! Rather than `isActive` method, I just want to access `fontSize`, `fontFamily`, etc value from selected node. Is that possible without using extension?

I've tried `editor.getAttributes('style')` however it returns empty object.

Thanks.

**Comment by vinicius-eidy** [comment]
I think it's impossible without using extensions, tiptap don't supports styling nodes in core.
According to: https://github.com/ueberdosis/tiptap/issues/64#issuecomment-442420681:
> Hey, I decided to not support _styling_ nodes in this core packages anymore.
